modules/social_features/social_group/modules/social_group_request/src/SocialGroupRequestConfigOverride.php

```diff
@@ -0,0 +1,716 @@
+<?php
+
+namespace Drupal\social_group_request;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\StringTranslation\TranslationInterface;
+
+/**
+ * Social Group request configuration override.
+ */
+class SocialGroupRequestConfigOverride implements ConfigFactoryOverrideInterface {
```

Comment:
When trying to enable the module I get an, could you please fix the trait / construct here:

PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Drupal\social_group_request\SocialGroupRequestConfigOverride::__construct(), 2 passed in /app/web/core/lib/Drupal/Component/DependencyInjection/Container.php on line 273 and exactly 3 expected in /app/web/profiles/contrib/social/modules/social_features/social_group/modules/social_group_request/src/SocialGroupRequestConfigOverride.php:43

Reply:
It happened because I was refactoring code blind :)

Already fixed. Check, please

Reply:
🙈